server/views/partials/offenderDetails.html

```diff
@@ -61,28 +61,28 @@
       {{ headerTitle('HDC Eligibility Date') }}
       {{ headerValue(dateConverter(data.details.sentence.homeDetentionCurfewEligibilityDate)) }}
     {% endif %}
-    {% if data.details.sentence.automaticReleaseDate %}
+    {% if data.details.sentence.automaticReleaseOverrideDate or data.details.sentence.automaticReleaseDate %}
       {{ headerTitle('Automatic Release Date') }}
-      {{ headerValue(dateConverter(data.details.sentence.automaticReleaseDate)) }}
+      {{ headerValue(dateConverter(data.details.sentence.automaticReleaseOverrideDate or data.details.sentence.automaticReleaseDate)) }}
     {% endif %}
-    {% if data.details.sentence.conditionalReleaseDate %}
+    {% if data.details.sentence.conditionalReleaseOverrideDate or data.details.sentence.conditionalReleaseDate %}
       {{ headerTitle('Conditional Release Date') }}
-      {{ headerValue(dateConverter(data.details.sentence.conditionalReleaseDate)) }}
+      {{ headerValue(dateConverter(data.details.sentence.conditionalReleaseOverrideDate or data.details.sentence.conditionalReleaseDate)) }}
     {% endif %}
     {% if data.details.sentence.paroleEligibilityDate %}
       {{ headerTitle('Parole Eligibility Date') }}
       {{ headerValue(dateConverter(data.details.sentence.paroleEligibilityDate)) }}
     {% endif %}
-    {% if data.details.sentence.nonParoleDate %}
+    {% if data.details.sentence.nonParoleOverrideDate or data.details.sentence.nonParoleDate %}
       {{ headerTitle('Non Parole Date') }}
-      {{ headerValue(dateConverter(data.details.sentence.nonParoleDate)) }}
+      {{ headerValue(dateConverter(data.details.sentence.nonParoleOverrideDate or data.details.sentence.nonParoleDate)) }}
     {% endif %}
```

Comment:
LGTM.  Is there any way of setting **data.details.sentence** as it's own variable and using something more concise rather than **data.details.sentence.ETC** every time?

Reply:
good idea

Reply:
Awesome.